ConsoleGameLibrary.sln

```diff
@@ -1,30 +1,50 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 11.00
-# Visual C# Express 2010
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This will break compatibility with developing in VS express 2010, this is probably OK but is worth evaluating.